include/effects/matrix/PatternAnimatedGIF.h

```diff
@@ -124,6 +124,13 @@ struct
     int             _offsetY   = 0;
```

Comment:
These changes allow animatged gif to draw to a canvas where scaling is required to fit.